seaworthy/utils.py

```diff
@@ -1,6 +1,2 @@
-def resource_name(name, namespace='test'):
-    return '{}_{}'.format(namespace, name)
-
-
 def output_lines(raw_output, encoding='utf-8'):
     return raw_output.decode(encoding).splitlines()
```

Comment:
Should we just move this to `logs.py` and get rid of `utils.py` entirely?

Reply:
Yeah I think that is a better place.. let me try squeeze that in.